### PR TITLE
Fullscreen control

### PR DESCRIPTION
**Pull request by alan-wu**

### CODE REVIEW

**Comment by alan-wu** [comment]
This also add the navigation control on flatmap and remove free spin from scaffoldvuer.

File: src/components/DialogToolbarContent.vue [hunk 1]
```diff
@@ -7,10 +7,11 @@
         </div>
       </div>
     </el-row>
-    <el-row class="icon-group" v-if="showIcons">
-      <el-button class="icon-transform" icon="el-icon-copy-document" size="medium" type="text" @click="maximise"></el-button>
-      <el-button icon="el-icon-remove-outline" size="medium" type="text" @click="minimise"></el-button>
-      <el-button icon="el-icon-close" size="medium" type="text" @click="close"></el-button>
+    <el-row class="icon-group" >
+      <el-button icon="el-icon-full-screen" size="medium" type="text" v-if="showFullscreenIcon" @click="onFullscreen"></el-button>
```
Comment (Tehsurfer) [inline src/components/DialogToolbarContent.vue:11]: One small thing, should full screen button stay in the top right?

Comment (Tehsurfer) [reply src/components/DialogToolbarContent.vue:11]: Actually don't worry I'll just accept this and create a new PR with fullscreen button always on the right

Comment (alan-wu) [reply src/components/DialogToolbarContent.vue:11]: I think the close button should be on the right but I am open to change it if that is agreed upon.

Comment (Tehsurfer) [reply src/components/DialogToolbarContent.vue:11]: Yeah it's not a biggie, can always change it if design finds it is better on the right.

Comment (Tehsurfer) [reply src/components/DialogToolbarContent.vue:11]: Happy to accept merge this in if that's what you are waiting for? But I don't have write access.

**Pull request merged**